.github/workflows/ci.yml

```diff
@@ -110,3 +112,39 @@ jobs:
         timeout_minutes: 15
         max_attempts: 2
         command: ./tools/test_monitor/run-tests.sh
+  flow-legacy-client-compatibility-test:
```

Comment:
Maybe use `localnet-test` as the name. This name is pretty specific to the refactoring use case and won't be accurate after the refactor PRs are complete.

Reply:
Indeed, we will have a different protocol.

Reply:
The issue was that I wanted to make sure that reviewers understand why we do not use a unit test as an integration test.
The status means to be end-to-end functional, and it can be presented to customers, if it passes this test.

Fix: https://github.com/GetElastech/flow-go/commit/5ad8e76874b0d9db1e92e87ac6f057cd53a4c1d9

Reply:
> Localnet Compatibility Tests With Legacy Client and Observer

what is the legacy client?

Reply:
The one shipped whithin brew for example: https://docs.onflow.org/flow-cli/install/

Reply:
How about
`Localnet Compatibility Tests With Flow-CLI Client and Observer`

Reply:
Fair enough.
Fix: https://github.com/GetElastech/flow-go/commit/76873e901fafc570c970ed69d8867be1823708d6
Thank you for the review.

Reply:
Fix: https://github.com/GetElastech/flow-go/commit/fd53d53a88882aaafb0980975a1178841bf38028